### DISCUSSION TITLE
`MultilineRawStringIndentation` not documented

### DISCUSSION THREAD

**Discussion by tinder-cesardiez** [detekt/detekt#5210] [Q&A] [answered]
Is it expected to not have documentation around `MultilineRawStringIndentation` on https://detekt.dev/docs/rules/style?

I'm asking because the rule might not be ready yet or might be coming on a future release, but it's present on the config file: https://github.com/detekt/detekt/blob/main/detekt-core/src/main/resources/default-detekt-config.yml#L608

**Comment by cortinico** [comment, marked as answer]
> Is it expected to not have documentation around `MultilineRawStringIndentation` on [detekt.dev/docs/rules/style](https://detekt.dev/docs/rules/style)?

We updated the docs to use versioning to solve questions like this one.

You can find docs about `MultilineRawStringIndentation` here:
https://detekt.dev/docs/next/rules/style
(in the `next` version).

If you check instead the latest released version instead:
https://detekt.dev/docs/rules/style
you won't find the rule (as in 1.21.0 the rule was not existing yet).

**Reply by tinder-cesardiez** [reply]
Oh, cool, thank you so much!
